analysis/import/SCMLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparser/SCMLogParser.kt

```diff
@@ -73,7 +73,8 @@ class SCMLogParser(private val input: InputStream = System.`in`,
                     "number_of_renames",
                     "range_of_weeks_with_commits",
                     "successive_weeks_of_commits",
-                    "weeks_with_commits"
+                    "weeks_with_commits",
+                    "highly_coupled_files"
```

Comment:
why don't we need `medium_coupled_files` here as well?

Reply:
*median_coupled_files ;) yes you are right, we can include this one here too. It's not strictly within the scope of the issue but as a boy-scout, it surely makes sense to calculate all metrics we can.